packages/api/src/external/carequality/command/cq-directory/rebuild-cq-directory-raw-sql.ts

```diff
@@ -24,8 +24,7 @@ const number_of_keys = keys.split(",").length;
 
 export async function insertCqDirectoryEntries(
   sequelize: Sequelize,
-  orgDataArray: CQDirectoryEntryData2[],
-  tableName: string
```

Comment:
Removed table name from parameters because:
- it's always the same table;
- other functions in this file already didn't have that parameter;
- the table name is defined in this file.